normandy/recipes/api/serializers.py

```diff
@@ -119,3 +119,26 @@ class Meta:
             'date_created',
             'recipe',
         ]
+
+
+class SignatureSerializer(serializers.ModelSerializer):
+    timestamp = serializers.DateTimeField(read_only=True)
+    signature = serializers.ReadOnlyField()
+    x5u = serializers.ReadOnlyField()
+    public_key = serializers.ReadOnlyField()
+
+    class Meta:
+        model = Signature
+        fields = ['timestamp', 'signature', 'x5u', 'public_key']
+
+
+class SignedRecipeSerializer(serializers.ModelSerializer):
+    signature = SignatureSerializer()
+    recipe = serializers.SerializerMethodField()
```

Comment:
The docs suggest you can just use `RecipeSerializer` as the field and get the same effect: http://www.django-rest-framework.org/api-guide/relations/#nested-relationships